pkg/wdk/primitives.go

```diff
@@ -23,18 +26,83 @@ func (d String5to2000Bytes) Validate() error {
 // Base64String is a string in base64 format
 type Base64String string
 
+// Validate checks if the string is valid a base64 string
+func (s Base64String) Validate() error {
+	// Step 1: Check if the string's length is divisible by 4 (Base64 requirement)
+	if len(s)%4 != 0 {
+		return fmt.Errorf("base64 string length must be divisible by 4")
+	}
+
+	// Step 2: Validate padding
+	paddingCount := strings.Count(string(s), "=")
+	if paddingCount > 2 || (paddingCount > 0 && !strings.HasSuffix(string(s), strings.Repeat("=", paddingCount))) {
+		return fmt.Errorf("invalid base64 padding")
+	}
```

Comment:
hmmm, wouldn't it be enough to just do it, this way:
```suggestion
	if strings.HasSuffix(string(s), "===") {
		return fmt.Errorf("invalid base64 padding")
	}
```

if it has padding of 3 or more "=" then it should have suffix of "===" so it looks like good enough check 🙂